src/document/types.ts

```diff
@@ -132,6 +132,8 @@ export type Operation<A extends Action = Action> = A & {
     skip: number;
     /** Error message for a failed action */
     error?: string;
+    /** Unique operation id */
+    id?: string;
```

Comment:
Can we make it required? Since we are assigning an id in the reducer if not provided

Reply:
I'm keeping this optional so i's compatible with old operations that have not an ID

Reply:
Ok, let's keep it and refactor later